.agents/skills/trigger-cost-savings/SKILL.md

```
@@ -0,0 +1,158 @@
+### Medium Impact
+2. **Add debounce to `sync-user-data`** — 847 runs/day, often triggered in bursts.
+   ```ts
+   debounce: { key: `user-${userId}`, delay: "5s" }
+   ```
+
+### Low Impact / Best Practices
+3. **Add `maxDuration` to `generate-report`** — No timeout configured.
+   ```ts
+   maxDuration: 300  // 5 minutes
+   ```
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)